src/wandbot/apps/zendesk/extract_by_type.py

```diff
@@ -0,0 +1,13 @@
+def discourseExt(description) -> str:
```

Comment:
I think these function names might need to be camel case, but I'll let @parambharat have the final word